### PR TITLE
Shorten and smooth the transfer/multi-task/few-shot section

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose

Addresses #110. In #109, it was suggested that any in-depth discussion of MultiPLIER and DeepProfile should be moved to a section that is intended to demonstrate that multiple concepts described throughout the paper can and should be used together to study rare diseases (e.g., regularization _AND_ prior knowledge/data). 

Moving the discussion of MultiPLIER and DeepProfile out of the prior data/knowledge section required a rewrite of the transfer and multi-task learning section to "briefly" explain these approaches and set up the section about putting it all together. 

**This pull request is that rewrite!** 🚀  Although, I'm not sure I've hit the mark with respect to _brevity_. I have removed and shortened some references/description of studies from the multi-task and few-shot learning paragraphs. But in the interest of improving the flow of this section of the manuscript, I've also _added some text_.

#### Is there anything that you want to discuss further?

What else can I cut? ✂️ 

#### Is the pull request ready for review?

Yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/05.prior-knowledge.md [hunk 1]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
 Transfer learning can be supervised (one or both of the source and target domains have labels), or unsupervised (both domains are unlabeled).
-Though there are multiple types of transfer learning, in a later section we will focus on feature-representation-transfer [@doi:10.1109/TKDE.2009.191].
+Though there are multiple types of transfer learning, in a later section we will focus in-depth on feature-representation-transfer [@doi:10.1109/TKDE.2009.191].
 Feature-representation-transfer approaches learn representations from the source domain and apply them to a target domain [@doi:10.1109/TKDE.2009.191].
-<!-- TODO: Add a motivating example that's not DeepProfile or MultiPLIER and ties into the first section that I propose to be about dimensionality reduction, etc. :D -->
+For example, low-dimensional representations can be learned from tumor transcriptomic data and transferred to describe patterns associated with genetic alterations in cell line data [@doi:10.1186/s13059-020-02021-3].  <!--TODO: anything need to be added/revised here once the dimensionality reduction section is rewritten to be more general? -->
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
For example, low-dimensional representations can be learned from tumor transcriptomic data and transferred to describe patterns associated with genetic alterations in cell line data [@doi:10.1186/s13059-020-02021-3].  
<!--TODO: anything need to be added/revised here once the dimensionality reduction section is rewritten to be more general? -->
```
Minor suggestion but splitting this out to a new line so it's less likely to be missed :)

File: content/05.prior-knowledge.md [hunk 2]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
 
 Transfer learning is an approach where a model trained for one task or domain (source domain) is applied to another, typically related task or domain (target domain).
 Transfer learning can be supervised (one or both of the source and target domains have labels), or unsupervised (both domains are unlabeled).
-Though there are multiple types of transfer learning, in a later section we will focus on feature-representation-transfer [@doi:10.1109/TKDE.2009.191].
+Though there are multiple types of transfer learning, in a later section we will focus in-depth on feature-representation-transfer [@doi:10.1109/TKDE.2009.191].
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
Though there are multiple types of transfer learning, in a later section we will focus in-depth on feature-representation-transfer.
```
Not sure if necessary to cite twice here

File: content/05.prior-knowledge.md [hunk 3]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
 Feature-representation-transfer approaches learn representations from the source domain and apply them to a target domain [@doi:10.1109/TKDE.2009.191].
-<!-- TODO: Add a motivating example that's not DeepProfile or MultiPLIER and ties into the first section that I propose to be about dimensionality reduction, etc. :D -->
+For example, low-dimensional representations can be learned from tumor transcriptomic data and transferred to describe patterns associated with genetic alterations in cell line data [@doi:10.1186/s13059-020-02021-3].  <!--TODO: anything need to be added/revised here once the dimensionality reduction section is rewritten to be more general? -->
 
 Where transfer learning can be supervised or unsupervised, the related approaches multitask and few-shot learning are forms of supervised learning that generally rely on deep neural networks.
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
Whereas transfer learning can be supervised or unsupervised, the related approaches multitask and few-shot learning are forms of supervised learning that generally rely on deep neural networks.
```

File: content/05.prior-knowledge.md [hunk 4]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
 Multitask learning is an approach where classifiers are learned for _related tasks_ at the same time using a shared representation [@doi:10.1023/A:1007379606734], where task refers to an individual prediction being made.
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
Multitask learning is an approach where classifiers are learned for _related individual predictions_ (tasks) at the same time using a shared representation [@doi:10.1023/A:1007379606734].
```

File: content/05.prior-knowledge.md [hunk 5]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
 Few-shot learning is the generalization of a model trained on related tasks to a new task with limited labeled data (e.g., the detection of a patient with a rare disease from a low number of examples of that rare disease).
-
-<!-- TODO: Shorten considerably -->
+Multitask and few-shot learning are comprised of a variety of approaches and architectures that are beyond this scope of this work (see [@arxiv:1706.05098; @arxiv:1707.08114v2] and [@arxiv:1904.05046v3] for an overview).
+We include brief descriptions of selected studies to illustrate potential uses and limitations of these approaches below.
 
 Multitask neural networks that predict multiple tasks simultaneously are generally thought to improve performance over models that make predictions for a single task by learning a shared representation and effectively being exposed to more training data than the single task case [@doi:10.1023/A:1007379606734; @arxiv:1606.08793].
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
Multitask neural networks (which predict multiple tasks simultaneously) are thought to improve performance over single task models by learning a shared representation, effectively being exposed to more training data than single task models [@doi:10.1023/A:1007379606734; @arxiv:1606.08793].
```

File: content/05.prior-knowledge.md [hunk 6]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
-In predicting phenotypes from EHR data, Ding et al. demonstrated that multitask neural networks outperformed single-task networks for predicting complex rare phenotypes but not common phenotypes [@arxiv:1808.03331].
-Liu et al. developed a method to train long short-term memory networks, a type of recurrent neural network, to predict mortality in rare diseases using EHR data as input [@arxiv:2004.05318v2].
-Their method, Ada-SiT (Adaptation to Similar Tasks), was specifically designed for many tasks with insufficient data and allowed for task similarity to be measured during training.
+Ding et al. also demonstrated that performance gains were context-dependent; multitask neural networks outperformed single-task networks for predicting complex rare phenotypes from EHR data, but not common phenotypes [@arxiv:1808.03331].
 
 In contrast, one-shot or few-shot learning relies on using prior knowledge to generalize to new prediction tasks where there are a low number of examples [@arxiv:1904.05046v3], where a distance metric is learned from input data and used to compare new examples for prediction [@doi:10.1021/acscentsci.6b00367].
```
Comment (allaway) [inline content/05.prior-knowledge.md:48]:
```suggestion
In contrast, one-shot or few-shot learning relies on using prior knowledge to generalize to new prediction tasks [@arxiv:1904.05046v3], where a distance metric is learned from input data and used to compare new examples for prediction [@doi:10.1021/acscentsci.6b00367].
```

File: content/05.prior-knowledge.md [hunk 7]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
-However, the authors' results suggested underperformance of one-shot learning methods relative to baseline random forest models when structural similarity could not be exploited and did not show support for generalization of models trained on very different contexts from the target task.
-Quellec et al. presented a few-shot learning approach for detecting rare pathologies in fundus photographs [@arxiv:1907.09449v3].
-The authors trained a convolutional neural network (CNN) to predict common pathologies, which tended to cluster similar conditions in feature space.
-The learned feature space was then used to train a probabilistic model for each rare pathology.
-This approach outperformed multitask learning, which suggests few-shot learning provides an advantage in contexts where predicting common conditions simultaneously results in a loss of performance [@arxiv:1907.09449v3].  
+The authors' results suggested underperformance of one-shot learning methods relative to baseline random forest models when structural similarity could not be exploited and did not show support for generalization of models trained on very different contexts from the target task.
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
The authors' results suggested that when structural similarity could not be exploited, one-shot learning methods underperform relative to baseline random forest models, and that models trained on very different contexts from the target task did not generalize well.
```

Comment (allaway) [reply content/05.prior-knowledge.md]: I thought my suggestion would be shorter, but doesn't look like it

File: content/05.prior-knowledge.md [hunk 8]
```diff
@@ -29,30 +29,26 @@ Methods that leverage shared features include transfer learning, multitask learn
...
+Quellec et al. presented a few-shot learning approach for detecting rare pathologies in fundus photographs [@arxiv:1907.09449v3] that outperformed multitask learning, which suggested few-shot learning provides an advantage in contexts where predicting common conditions simultaneously results in a loss of performance [@arxiv:1907.09449v3].
```
Comment (allaway) [inline content/05.prior-knowledge.md]:
```suggestion
Quellec et al. presented a few-shot learning approach for detecting rare pathologies in fundus photographs that outperformed multitask learning, suggesting that few-shot learning provides an advantage in contexts where predicting common conditions simultaneously results in a loss of performance [@arxiv:1907.09449v3].  
```

**Review by allaway — APPROVED** [review]
I tried to edit it down and made a few suggestions that reduces the length marginally, but am approving since the suggestions I made don't appear to be that helpful in reducing the length :)

**Comment by jaybee84** [comment]
is this PR ready to be merged @jaclyn-taroni ?

**Comment by allaway** [comment]
Just bumping this up again - once we've merged this and #118, I'll take another stab at my issues and the rest of the doc.

**Comment by jaclyn-taroni** [comment]
@allaway I don't think I addressed all your comments, so if you are happy with where this is at currently please feel free to merge!

**Pull request merged**